Public/Src/App/Bxl/AzureDevOpsListener.cs

```diff
@@ -0,0 +1,212 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Diagnostics.ContractsLight;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using BuildXL.Utilities.Collections;
+using BuildXL.Utilities.Instrumentation.Common;
+using BuildXL.Utilities.Tracing;
+using BuildXL.ViewModel;
+
+namespace BuildXL
+{
+    /// <summary>
+    /// This event listener should only be hooked up with AzureDevOps optimzied UI is requested
+    /// It tracks data that is not easily captured at the site where the information is emitted because
+    /// it is too hard to track through. It is most ideal to capture data at the site of collections.
+    /// But as an example the cache miss analyzer is many levels of indirection deep and passing
+    /// the BuildSummary model down would be way to cumbersome.

```

Comment:
nit: same comment.. could you pls rephrase this sentence? I'm not sure that I'm getting it.